sdk/jwt.go

```diff
@@ -95,6 +91,14 @@ func (j jwtClaims) Validate() error {
 	if j.M2MProfile.Value != nil && j.AgentProfile.Value != nil {
 		return errors.New("expected either 'm2mProfile' or 'agentProfile' to be set in JWT claims, but both were found")
 	}
+	if j.ExpiresAt == nil || j.ExpiresAt.IsZero() {
+		return errTokenMissingExpiryClaim
+	}
+	fmt.Println(j.ExpiresAt.String())
+	fmt.Println(time.Now().String())
+	if time.Now().After((j.ExpiresAt).Add(jwtLeeway)) {
```

Comment:
```suggestion
	if time.Now().Add(-jwtLeeway).After(j.ExpiresAt.Time) {
```



This adds leeway to the expiration time, making tokens valid longer. It should subtract the leeway from current time or add it to the token's issued time instead.